### PR TITLE
added formality to style and exaggerating is un_bias

### PR DESCRIPTION
**Pull request by lnazarenko**
- added subcategory "formality" to the style. Now "Sehr geehrte/ Sehr geehrter" are triggers.
- moved "exaggerating" from "style" to "un_bias" (according to notion page) and adjusted tests correspondently

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1195,7 +1195,7 @@ def GenderedDenomAnalysis(config: Config, lang, full_text, tokens, df):
 def ExaggeratingWordsSentences(
     config: Config, lang, full_text, tokens, df, df_sentences
 ):
-    category = "style"
+    category = "unconscious_bias"
```
Comment (lsmith77) [inline app/main.py:1198]: why did you make this change?

from what I can see its in style?
https://www.notion.so/witty-works/exaggerating-de1a59b1775248ada4239fc3eff15760

Comment (lnazarenko) [reply app/main.py:1198]: The link that you shared clearly shows that "Exaggerating" is in uncon. bias category now. Please have a look carefully, the color shema is marked as style but category in uncon. bias. You can check this also in the witty.rules sheet.

Comment (lsmith77) [reply app/main.py:1198]: ah .. sorry for missing that

Comment (lsmith77) [reply app/main.py:1198]: with this PR this kind of thing should not automatically stay in sync https://github.com/witty-works/nlp_api/pull/269

**Pull request merged**